neuralpredictors/layers/readouts/factorized.py

```diff
@@ -19,12 +23,31 @@ def __init__(
         init_noise=1e-3,
         constrain_pos=False,
         positive_weights=False,
+        positive_spatial=True,
         shared_features=None,
         mean_activity=None,
         spatial_and_feature_reg_weight=None,
-        gamma_readout=None,  # depricated, use feature_reg_weight instead
+        gamma_readout=None,
         **kwargs,
     ):
+        """
+
+        Args:
+            in_shape: batch, channels, height, width (batch could be arbitrary)
+            outdims: number of neurons to predict
+            bias: if True, bias is used
+            normalize: if True, normalizes mask using by l2 norm
+            init_noise: the std for readout  initialisation
+            constrain_pos: if True, negative values in the neuronal features are turned into 0
+            positive_weights: if True, negative values in the mask are turned into 0
+            positive_spatial: if True, mask values are restricted to be positive by taking absolute values
+            shared_features: if True, uses a copy of the features from somewhere else
+            mean_activity: the mean for readout  initialisation
+            spatial_and_feature_reg_weight: lagrange multiplier (constant) for L1 penalty,
+                the bigger the number, the stronger the penalty
+            gamma_readout: depricated, use feature_reg_weight instead
```

Comment:
```suggestion
            gamma_readout: depricated, use spatial_and_feature_reg_weight instead
```
We could think of additionally raising a user warning by `warnings.warn`

Reply:
I thing this might be a different PR probably because we might change it consistently everywhere in readouts (not only here)